config/sys.config.src

```diff
@@ -0,0 +1,123 @@
+%% -*- erlang -*-
```

Comment:
what is this?

Reply:
@evanmcc Ignore that added by mistake.  Removed already but I must not have pushed.